gcsfs/dask_link.py

```diff
@@ -118,7 +120,6 @@ def walk(self, path, refresh=False):
     dask.bytes.core._filesystems['gs'] = DaskGCSFileSystem
 
 
-try:
+# Register to dask if dask is installed
+if 'dask' in sys.modules:
     register()
-except ImportError as e:
-    print(e)
```

Comment:
I suggest the easiest thing to do here is to change the print to a logger.debug statement, so it doesn't normally show unless the user enables debug logging.

Reply:
Hi Martin,

Thanks for pointing it out. Yes, logger.debug will be much easier. Thanks for the suggestion.

Best,

Qibin